pkg/mimedb/util/gen-db.go

```diff
@@ -127,18 +129,18 @@ func main() {
 	// as key.
 	mDB, err := convertDB(os.Args[1])
 	if err != nil {
-		panic(err)
+		logger.FatalIf(err, "")
```

Comment:
Can this file be modifed? It says `DO NOT EDIT THIS FILE`

Reply:
That is correct. I thought it is not part of this PR. I'll revert it back.
Done.